include/ofi_list.h

```diff
@@ -194,6 +194,124 @@ static inline void dlist_splice_tail(struct dlist_entry *head,
 	dlist_splice_head(head->prev, to_splice);
 }
 
+/*
+ * Multi-threaded Double-linked list
+ */
+struct dlist_mt {
```

Comment:
Do we like mt for multi-threaded or ts for thread safe better?

Reply:
+1 for thread safe